pkg/blobfuse-proxy/init.sh

```diff
@@ -33,25 +33,50 @@ then
   $HOST_CMD rm -f /etc/packages-microsoft-prod.deb
 fi
 
-if [ ! -f "/host/usr/bin/blobfuse-proxy" ];then
+updateBlobfuseProxy="true"
+if [ -f "/host/usr/bin/blobfuse-proxy" ];then
+  old=$(sha256sum /host/usr/bin/blobfuse-proxy | awk '{print $1}')
+  new=$(sha256sum /blobfuse-proxy/blobfuse-proxy | awk '{print $1}')
+  if [ "$old" = "$new" ];then
+    updateBlobfuseProxy="false"
+    echo "no need to update blobfuse-proxy"
+  else
+    rm -rf /host/usr/bin/blobfuse-proxy
```

Comment:
since it would be overwritten later, no need to do this?

Reply:
fixed